.github/workflows/tests.yml

```
@@ -237,19 +237,32 @@ jobs:
         if: always()
         run: |
           touch bottles/steps_output.txt
-          cat bottles/steps_output.txt
+
+          echo "# Build failure summary on ${{ matrix.runner }}" >> "$GITHUB_STEP_SUMMARY"
```

```suggestion
          echo "#### Build failure summary on ${{ matrix.runner }}" >> "$GITHUB_STEP_SUMMARY"
```

or similar to make text less huge